src/core/js/views/questionView.js

```diff
@@ -39,7 +39,19 @@ define([
       this._runModelCompatibleFunction('setupDefaultSettings');
       // Blank method for setting up questions before rendering
       this.setupQuestion();
+    }
+
+    /** @type {boolean} */
+    get canShowMarking() {
```

Comment:
Move to `QuestionModel` as not accessible in `ButtonsView` my mistake. I'm clearly not thinking properly.